Sources/AloeStackView/AloeStackView.swift

```diff
@@ -20,506 +20,531 @@ import UIKit
  * power of Auto Layout.
  */
 open class AloeStackView: UIScrollView {
+    // MARK: Lifecycle
```

Comment:
Please revert all of these style changes.

Reply:
Yes, I just committed again.